snap/snapcraft.yaml

```diff
@@ -31,8 +31,17 @@ parts:
       sed -e '1c#!/usr/bin/env python3' -i $CRAFT_PART_INSTALL/snap/blanket/current/usr/bin/blanket
     organize:
       snap/blanket/current: .
+
+  gpu-2404:
```

Comment:
This is actually not necessary. Most things here are done from the extension.

Reply:
Without this, I get the following linter warnings when building the snap:

```
Ignoring invalid url 'email_AT_rafaelmardojai.com' in update_contact from appstream metadata.
Lint warnings:
- library: libEGL_mesa.so.0: unused library 'usr/lib/x86_64-linux-gnu/libEGL_mesa.so.0.0.0'. (https://snapcraft.io/docs/linters-library)
- library: libGLX_mesa.so.0: unused library 'usr/lib/x86_64-linux-gnu/libGLX_mesa.so.0.0.0'. (https://snapcraft.io/docs/linters-library)
- library: libcairo-gobject.so.2: unused library 'usr/lib/x86_64-linux-gnu/libcairo-gobject.so.2.11800.0'. (https://snapcraft.io/docs/linters-library)
- library: libdconf.so.1: unused library 'usr/lib/x86_64-linux-gnu/libdconf.so.1.0.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstadaptivedemux-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstadaptivedemux-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstanalytics-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstanalytics-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstbadaudio-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstbadaudio-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstcheck-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstcheck-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstcontroller-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstcontroller-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstcuda-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstcuda-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstdxva-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstdxva-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstfft-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstfft-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstinsertbin-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstinsertbin-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstisoff-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstisoff-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstmpegts-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstmpegts-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstmse-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstmse-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstnet-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstnet-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstplayer-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstplayer-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstriff-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstriff-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstrtsp-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstrtsp-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstsctp-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstsctp-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgsttranscoder-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgsttranscoder-1.0.so.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstva-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstva-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstvulkan-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstvulkan-1.0.so.0.2402.0'. (https://snapcraft.io/docs/li
nters-library)
- library: libgstwayland-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstwayland-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libgstwebrtcnice-1.0.so.0: unused library 'usr/lib/x86_64-linux-gnu/libgstwebrtcnice-1.0.so.0.2402.0'. (https://snapcraft.io/docs/linters-library)
- library: libicuio.so.74: unused library 'usr/lib/x86_64-linux-gnu/libicuio.so.74.2'. (https://snapcraft.io/docs/linters-library)
- library: libicutest.so.74: unused library 'usr/lib/x86_64-linux-gnu/libicutest.so.74.2'. (https://snapcraft.io/docs/linters-library)
- library: liborc-test-0.4.so.0: unused library 'usr/lib/x86_64-linux-gnu/liborc-test-0.4.so.0.38.0'. (https://snapcraft.io/docs/linters-library)
- library: libunwind-coredump.so.0: unused library 'usr/lib/x86_64-linux-gnu/libunwind-coredump.so.0.0.0'. (https://snapcraft.io/docs/linters-library)
- library: libunwind-ptrace.so.0: unused library 'usr/lib/x86_64-linux-gnu/libunwind-ptrace.so.0.0.0'. (https://snapcraft.io/docs/linters-library)
- library: libunwind-x86_64.so.8: unused library 'usr/lib/x86_64-linux-gnu/libunwind-x86_64.so.8.0.1'. (https://snapcraft.io/docs/linters-library)
Packed blanket_0.7.0_amd64.snap
```

Reply:
Because you're staging the gpu related libraries somewhere in the snap. See, the error log, libGLex unused library. That library is also in the gpu provider. You basically need to cleanup the snap and be critical on what to stage and what not.

Reply:
How do you cleanup the snap?  I ran `snapcraft clean` and then `snapcraft` to build it and I got those errors.

Reply:
I think the `libgstreamer-plugins-bad1.0-0` is pulling in the `libEGL` as an indirect dependency... But the package requires this, so I'm not sure what to do about removing that library.